jssg/models.py

```diff
@@ -335,11 +335,18 @@ class PostList :
     def __init__(self, category = "", page = 1) -> None:
         self.category = category
         self.page = page
-        self.posts_by_page = settings.JFME_NUMBER_OF_POSTS_BY_PAGE
-        if self.category == "" :
-            self.nb_pages = ceil(len(list(Post.load_glob(all=True))) / self.posts_by_page) # number of posts / number of posts by page
+        
+        if category == "" :
+            nb_posts = len(list(Post.load_glob(all = True)))
         else :
-            self.nb_pages = ceil(len(list(filter(lambda p: p.metadata["category"] == self.category, Post.load_glob(all=True)))) / self.posts_by_page) # number of posts of the category / number of posts by page
+            nb_posts = len(list(filter(lambda p: p.metadata["category"] == self.category, Post.load_glob(all=True))))
+
+        if hasattr(settings, "JFME_NUMBER_OF_POSTS_BY_PAGE") :
```

Comment:
It's not a common way to disable a feature to remove the parameter ; the right way is to have a default value.

I suggest here to replace

```
if hasattr(settings, "JFME_NUMBER_OF_POSTS_BY_PAGE") :
```

With

```
if settings.JFME_NUMBER_OF_POSTS_BY_PAGE <= 0:
```

This way the parameter is always available; this is a common pattern to set to 0 or -1 an attribute in order to tell "this is unlimited"